cmd/cmd.go

```diff
@@ -22,6 +22,7 @@ type ConfigurableLogger interface {
 	WantVerbose(f bool)
 }
 
+//nolint:staticcheck
```

Comment:
Based on this: https://golangci-lint.run/usage/false-positives/#nolint
we can have the `nolint:staticcheck` exclusion at the beginning of the function, and remove a bit of the clutter from this method (before we remove them)